matlab/src/feather_writer.h

```diff
@@ -17,40 +17,36 @@
 
 #pragma once
 
-#include <memory>
-#include <string>
-
 #include <arrow/ipc/feather.h>
 #include <arrow/status.h>
 #include <arrow/type.h>
-
 #include <matrix.h>
 
+#include <memory>
+#include <string>
```

Comment:
Is there a reason to move these here?

Reply:
Nope. See my comment below. I'll move them back to the top.